files/nginx.sh

```diff
@@ -11,6 +11,12 @@ if ! grep -q "worker_rlimit_nofile    20000;" /usr/local/etc/nginx/nginx.conf; t
   gsed -i '/worker_processes  auto;/a worker_rlimit_nofile    20000;' /usr/local/etc/nginx/nginx.conf
 fi
 
+echo "Set client_max_body_size to 512M"
+
+if ! grep -q "client_max_body_size 512M;" /usr/local/etc/nginx/nginx.conf; then
+  gsed -i 'http {/a client_max_body_size 512M;' /usr/local/etc/nginx/nginx.conf
+fi
+
```

Comment:
https://i.imgur.com/DjrTV0T.png
we are using `conf.d` for these parameters.

Reply:
maybe for this option already let it be a script, but for the future I understood